build/build.py

```diff
@@ -185,30 +185,31 @@ def get_bazel_paths(bazel_path_flag):
 
 def get_bazel_path(bazel_path_flag):
   """Returns the path to a Bazel binary, downloading Bazel if not found. Also,
-  it checks Bazel's version at lease newer than 2.0.0.
+  checks Bazel's version is at least newer than 4.1.0.
 
-  NOTE Manual version check is reasonably only for bazel < 2.0.0. Newer bazel
-  releases performs version check against .bazelversion (see for details
+  A manual version check is needed only for really old bazel versions.
+  Newer bazel releases perform their own version check against .bazelversion
+  (see for details
   https://blog.bazel.build/2019/12/19/bazel-2.0.html#other-important-changes).
   """
   for path in filter(None, get_bazel_paths(bazel_path_flag)):
-    if check_bazel_version(path):
-      return path
+    version = get_bazel_version(path)
+    if version is not None and version >= (4, 1, 0):
+      return path, ".".join(map(str, version))
 
-  print("Cannot find or download bazel. Please install bazel.")
+  print("Cannot find or download bazel >= 4.1.0. Please install bazel.")
```

Comment:
nit: Please install bazel >= 4.1.0?

I know you say it before but someone might skip it, so its good to add the version that you are asking to install.

Reply:
I reworded it a bit to avoid needing to repeat the version number.